.gitignore

```diff
@@ -15,3 +15,9 @@ Localize/out/
 /packages/service/resources/i18n/*
 web/dev-server/resources/i18n/*
 desktop/resources/i18n/*
+/packages/common/src/ui-common/localization/generated/*
```

Comment:
Suggestion: Let's flip this around so `generated` is a top-level folder. This may have been a better place to put the generated .json files too now that I think about it...